static/css/components/header-bar.less

```
@@ -287,6 +287,28 @@
       height: 30px;
     }
 
+    .mr-notifications {
+      position: absolute;
+      background: @link-blue;
```

```suggestion
      background: @primary-blue;
```
Change color to match the app drawer badge's color.